docs/docs/configuration/configuring-superset.mdx

```diff
@@ -87,23 +87,15 @@ can be altered in your local `superset_config.py`. Administrators will want to r
 to understand what can be configured locally as well as the default values in place.
 
 Since `superset_config.py` acts as a Flask configuration module, it can be used to alter the
-settings Flask itself, as well as Flask extensions that Superset bundles like
+settings of Flask itself, as well as Flask extensions that Superset bundles like
 `flask-wtf`, `flask-caching`, `flask-migrate`,
 and `flask-appbuilder`. Each one of these extensions offers intricate configurability.
 Flask App Builder, the web framework used by Superset, also offers many
 configuration settings. Please consult the
 [Flask App Builder Documentation](https://flask-appbuilder.readthedocs.org/en/latest/config.html)
 for more information on how to configure it.
 
-You'll want to change:
-
-- `SECRET_KEY`: to a long random string
-- `SQLALCHEMY_DATABASE_URI`: that by default points to sqlite database located at
-  ~/.superset/superset.db
-
-```
-WTF_CSRF_EXEMPT_LIST = [‘’]
```

Comment:
This line lingered accidentally after other content was deleted.  The same variable already appears higher above on this page.